certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -1260,144 +1387,70 @@
-                        "description": "entityId"
+                        "description": "The 3 digit fref exchange code for the primary exchange of the security"
                       },
-                      "lookupUrl": {
+					  "< * >": {
                         "type": "string",
-                        "description": "lookupUrl"
+                        "description": "The Key in the Key-Value pair is the name of the requested outputSymbolType. The Value in the Key-Value is the requestId translated to the outputSymbolType."
                       },
-                      "associatedOrgs": {
-                        "type": "array",
-                        "items": {
-                          "type": "object",
-                          "properties": {
-                            "name": {
-                              "type": "string",
-                              "description": "name"
-                            },
-                            "entityId": {
-                              "type": "string",
-                              "description": "entityId"
-                            },
-                            "lookupUrl": {
-                              "type": "string",
-                              "description": "lookupUrl"
-                            }
-                          }
-                        },
-                        "description": "associatedOrgs"
-                      }
                     }
                   },
-                  "description": "entities"
+                  "description": "Response object for Identifier Resolution endpoint."
                 }
               }
             }
           }
         },
-        "summary": "NER - Entities",
-        "operationId": "ner-entities",
+        "summary": "Symbology API",
+        "description": "Translate Entity ID and market security symbols into FactSet Permanent Security Identifiers.",
+        "operationId": "get-symbology",
         "parameters": [
           {
-            "name": "body",
-            "in": "body",
+            "name": "ids",
+            "in": "query",
             "required": false,
-            "schema": {
-              "type": "object",
-              "properties": {
-                "data": {
-                  "type": "object",
-                  "properties": {
-                    "enableIdLookup": {
-                      "type": "boolean",
-                      "description": "enableIdLookup"
-                    },
-                    "filterEntities": {
-                      "type": "boolean",
-                      "description": "filterEntities"
-                    },
-                    "text": {
-                      "type": "string",
-                      "description": "text"
-                    }
-                  },
-                  "description": "data"
-                }
-              }
-            }
+            "type": "string"
```

Comment:
@KyleVErwin 'x-ms-summary' and the 'description' property is required. Please add both to the parameter.